layouts/partials/stylesheets.html

```diff
@@ -1,10 +1,10 @@
 {{ $css := resources.Get "css/main.css" | minify | fingerprint "sha512" }}
-<link rel="stylesheet" href="{{ $css.RelPermalink }}" integrity="{{ $css.Data.Integrity }}">
+<link rel="stylesheet" href="{{ $css.Permalink }}" integrity="{{ $css.Data.Integrity }}">
```

Comment:
Unfortunately this was the only way to make this work reliably with the `baseUrl` being `a.b.c/d`

Reply:
When I tested it with a baseURL ending in `/spectral/`, hugo automatically made the RelPermalinks begin with `/spectral/`. Is there a case where that is not sufficient?

Reply:
When I try it locally it works too - for some reason it didn't, when I used the GH Workflow. Maybe let's just drop the change for now.